### ISSUE TITLE
Re-open request 302 redirect when using Laravel's Form Request Validation #148

### ISSUE THREAD

**Issue by simplyrupin** [fruitcake/laravel-cors#280]
Hi,

This issue is for form request validation (after installing laravel-cors).

I tried the solution mentioned in #148 but it didn't solve my problem.

When validation errors are thrown for simple form request validations, I get a 302 with the following "Requests to the server have been blocked by an extension."

When using post validation functions ($validator->after()) the rules are skipped and the post validation method executes as usual. Which is a massive issue right now since post validation methods aren't expected to run if validation fails. 

Any workarounds or fixes for this?

Thanks

**Comment by ArtemMolotov** [comment]
I think, I have same problem. Please, see my comments in other issue (239).

**Comment by ArtemMolotov** [comment]
#278
@acacha: 
> Ok I've found the problem (in my case). As seen in docs:
> 
> https://github.com/barryvdh/laravel-cors#common-problems-and-errors-pre-laravel-53
> 
> "When an error occurs, the middleware isn't run completely."
> 
> So any error could returns a CORS problem because middleware CORS doesn't runs oks. This but don't mean anything is wrong with CORS it could be any other problem.
> 
> In my case CORS doesn't works because I'm dumping (for debugging reasons) values to api output so I provoke a Headers already sent error.

**Comment by barryvdh** [comment]
I'm closing this due to inactivity. If you still encounter this issue on the latest version (2.0.1 at the moment), please open a new issue with detailed information.
